server/db/schema/class_enrollments.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS class_enrollments (
+    id SERIAL PRIMARY KEY,
+    student_id INT NOT NULL,
```

Comment:
In general prefer INTEGER for consistency -- I won't be enforcing this since functionally int is the same.

Reply:
Got it, I will be aware of that rule in the future.